### DISCUSSION TITLE
Name spoofed github package registry gchr.io

### DISCUSSION THREAD

**Discussion by pettermk** [community/community#62655] [Code Security] [answered]
### Select Topic Area

Question

### Body

Hi,

Today I inadvertently did a

```
docker login gchr.io
```
and entered my username and PAT token which had just been created with the package:write scope. Although it did say login succeeded, I immediately discovered my error and deleted the token.

It is quite easy to mix up gchr.io and ghcr.io. I did not find any info about this apparently spoofed registry and I was wondering what sort of risk I had put myself into. I don't know if it is possible for github to take action against the registry, but I would feel better if it was not so easy to make this mistake.

BR,
Petter

**Comment by radusuciu** [comment, marked as answer]
Seems like the domain is no longer registered, were I GH, I would snap it up...

**Reply by pettermk** [reply]
Confirmed, domain is now up for grabs, good news :+1:
